Gemfile

```diff
@@ -1,3 +1,3 @@
 source "https://rubygems.org"
 
-gem "bugsnag-maze-runner", git: 'https://github.com/bugsnag/maze-runner', :branch => 'v1'
+gem "bugsnag-maze-runner", git: 'https://github.com/bugsnag/maze-runner', branch: 'fix-build-api-steps'
```

Comment:
This needs to be changed back once https://github.com/bugsnag/maze-runner/pull/125 has been merged/released.  I'm going to address a couple of rubocop things there first.

Reply:
Any reason not to merge and release #125 now?

Reply:
But also, how do we want to track Maze Runner from BAGP when it's not using the Docker image?  Just pinning it to a release (tag) and not worrying about trying to track the latest at this stage?

Reply:
Pretty much.  We should pin it to a version or branch tag and only update when necessary.